### PR TITLE
Initialized docker environment

### LINKED ISSUE #67: Throw error whenever unable to execute
The current code generated blank O/P files instead of throwing an error and exiting code.

### PR DESCRIPTION
**Pull request by mark-nakachon**
### Checklist:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you successfully ran tests with your changes locally?
* [x] Have you added an explanation of what your changes do and why you'd like us to include them?

### Issues Solved

Fixes #67 

Initialized a Docker environment

### CODE REVIEW

**Comment by aashutoshrathi** [comment]
You are pointing to wrong issues in the description please fix that.

File: views/components/header.ejs [hunk 1]
```diff
@@ -39,4 +39,4 @@
                 </ul>
             </div>
         </div>
-    </nav>
+    </nav>
```
Comment (aashutoshrathi) [inline views/components/header.ejs]: You can just quick fix the newline issues here

Comment (mark-nakachon) [reply views/components/header.ejs]: Added a new line merge this PR

File: package.json [hunk 2]
```diff
@@ -24,4 +24,4 @@
 		"type": "git",
 		"url": "https://git.io/tcgen"
 	}
-}
+
```
Comment (aashutoshrathi) [inline package.json]: I request you to please not mess up with package.json unnecessarily.
You removed the last brace. 😕

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+
+WORKDIR /usr/src/hackerrank-testcase-generator
+
+COPY ./ ./
+
+RUN npm install
+
+
+CMD ["sh"]
```
Comment (aashutoshrathi) [inline Dockerfile]: Make it
`CMD ["npm", "start"]`

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,11 @@
+FROM node:alpine
+
+
+WORKDIR /usr/src/hackerrank-testcase-generator
+
+COPY ./ ./
+
+RUN npm install
+
+
```
Comment (aashutoshrathi) [inline Dockerfile]: And these 2 blank lines don't make much sense, make it 1.

Comment (aashutoshrathi) [inline Dockerfile]: Also, add env vars too.
`ENV MongoURI here`

Comment (mark-nakachon) [reply Dockerfile]: Updated the Docker file

**Comment by mark-nakachon** [comment]
I Updated README.md please review and close pr

File: README.md [hunk 5]
```diff
@@ -53,6 +53,11 @@ C / C++ / Java / Python | [TC Generator](/tc_generator/tc_gen.py) |
 
 ![Demo](demo2.gif)
 
+### How to Setup a Server?
```
Comment (aashutoshrathi) [inline README.md]: ### Setup using Docker

**Comment by aashutoshrathi** [comment]
@mark-nakachon Will merge after this.

**Comment by mark-nakachon** [comment]
Made a quick change

File: README.md [hunk 6]
```diff
@@ -53,6 +53,13 @@ C / C++ / Java / Python | [TC Generator](/tc_generator/tc_gen.py) |
 
 ![Demo](demo2.gif)
 
+### Setup using Docker?
```
Comment (aashutoshrathi) [inline README.md]: Please remove `?` ASAP.

**Comment by mark-nakachon** [comment]
Removed question mark

**Pull request merged**